toqito/state_metrics/fidelity_of_separability.py

```diff
@@ -0,0 +1,203 @@
+"""Add function for fidelity of separability as defined in [Phil23]_.
+
+The constrainsts for this function are positive partial transpose (PPT)
+& k-extendible states.
+"""
+
+import picos
+import numpy as np
+
+from toqito.perms import symmetric_projection
+from toqito.state_props import is_pure
+from toqito.state_props.is_separable import is_separable
```

Comment:
Think you can get away with just:

```suggestion
from toqito.state_props import is_separable

```

Reply:
Same as above, there was a build failure issue.

Reply:
@vprusso I'll go ahead & accept this. Seems `is_separable` `not callable` issue might have been fixed with the linked StackExchange answer. 

I tested the `is_mutually_orthogonal` without specifying the full path but I got the same errors.